packages/create-sheriff-config/tsup.config.json

```diff
@@ -3,5 +3,12 @@
   "format": "esm",
   "splitting": false,
   "sourcemap": false,
-  "clean": true
+  "clean": true,
+  "target": "node20",
+  "skipNodeModulesBundle": true,
+  "noExternal": ["@sherifforg/constants"],
```

Comment:
This is great, but in tandem we should also revert the old fix that i did, meaning bringign back this dep to the regular deps instead of devdeps